### ISSUE TITLE
Terraform aws_security_group false negative

### ISSUE THREAD

**Issue by konstruktoid** [Checkmarx/kics#5183]
Sorry for not submitting a PR, Rego learning in progress.

### Expected Behavior

```console
File: terraform/main.tf
Line 56
Expected: 'aws_security_group[cowrie]' is used
Found: 'aws_security_group[cowrie]' is used
```

### Actual Behavior

```console
File: terraform/main.tf
Line 56
Expected: 'aws_security_group[cowrie]' is used
Found: 'aws_security_group[cowrie]' is not used
```

```console
Security Group Not Used, Severity: INFO, Results: 1
Description: Security group must be used or not declared
Platform: Terraform

	[1]: terraform/main.tf:56

		055:
		056: resource "aws_security_group" "cowrie" {
		057:   name        = "CowrieSSH"

```

```sh
$ terraform show -json | jq '.values.root_module.resources[0].values.security_groups'
[
  "CowrieSSH",
  "default"
]
```

https://github.com/konstruktoid/ansible-cowrie-rootless/blob/main/aws/main.tf#L56 is used at https://github.com/konstruktoid/ansible-cowrie-rootless/blob/main/aws/main.tf#L86 as a variable (https://github.com/konstruktoid/ansible-cowrie-rootless/blob/main/aws/main.tf#L32).

### Specifications

Tested with 1.5.1 (https://github.com/Checkmarx/homebrew-tap/issues/1) and 1.5.5 (Docker).

**Comment by cxAndreFelicidade** [comment]
Hello Thomas, this is indeed a false positive, thank you for being so attentive! I hope this [PR](https://github.com/Checkmarx/kics/pull/5208) fixes your issue. Also, do you mind if we use the sample you provided as a negative sample?

**Comment by konstruktoid** [comment]
Thanks @cxAndreFelicidade and you're of course welcome to use the sample.
